### DISCUSSION TITLE
Upgrade TimescaleDB Extension on Hosted Instance

### DISCUSSION THREAD

**Discussion by erichosick** [supabase/supabase#11060] [Questions] [answered]
I have a hosted pro-tier instance that I enabled TimescaleDB on. The version is 2.6.1. How do I get this instance to use a more recent version of TimeScaleDB?

I've tried cycling the instance, removing and adding TimescaleDB again.

I had an older server instance (non-pro) that I enabled TimescaleDB on today (today). That instance is using a newer version of TimeScaleDB.

**Comment by GaryAustin1** [comment]
You might try to pause and restore as that tends to bring in the latest images.  

Edit:
But, I don't know if the extensions get updated as the Postgres version usually does not and the extensions may be tied to that image.  You could check version of Postgres with `select version()` in the SQL editor and if the Postgres version is newer on a the one you want, then you probably have to upgrade/migrate to a more recent instance, if the pause/restore did not do it.

**Comment by erichosick** [comment, marked as answer]
For pro, there doesn't seem to be a pause option: only a restart option:

![Screen Shot 2022-12-17 at 8 37 09 AM](https://user-images.githubusercontent.com/295228/208252040-53598149-d977-40c8-9cb4-82308bf8dc99.png)


I did a restore from a backup, which didn't change anything with regards to versions.

Reading the docs https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects implies that an upgrade is done "When you pause and restore a project, the restored database includes the latest features."

**Reply by GaryAustin1** [reply]
You probably need to contact support then.  I don't have a pro project, but I've seen that mentioned before (no pause) and a backup restore is not going to do it.
